### PR TITLE
Bump PandcGUI to 1.1

### PR DESCRIPTION
**Pull request by Ombrelin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40088)

### CODE REVIEW

File: manifests/o/Ombrelin/PandocGui/1.1/Ombrelin.PandocGui.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS0.5-1-22000-282
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Ombrelin.PandocGui
+PackageVersion: "1.1"
```
Comment (Trenly) [inline manifests/o/Ombrelin/PandocGui/1.1/Ombrelin.PandocGui.installer.yaml:5]:
```suggestion
PackageVersion: "1.1.35"
```

File: manifests/o/Ombrelin/PandocGui/1.1/Ombrelin.PandocGui.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS0.5-1-22000-282
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Ombrelin.PandocGui
+PackageVersion: "1.1"
```
Comment (Trenly) [inline manifests/o/Ombrelin/PandocGui/1.1/Ombrelin.PandocGui.locale.en-US.yaml:5]:
```suggestion
PackageVersion: "1.1.35"
```

File: manifests/o/Ombrelin/PandocGui/1.1/Ombrelin.PandocGui.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS0.5-1-22000-282
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Ombrelin.PandocGui
+PackageVersion: "1.1"
```
Comment (Trenly) [inline manifests/o/Ombrelin/PandocGui/1.1/Ombrelin.PandocGui.yaml:5]:
```suggestion
PackageVersion: "1.1.35"
```

**Review by Trenly — CHANGES_REQUESTED** [review]
The version must match what is in Control Panel, otherwise users will be stuck in an upgrade loop. 
Note that if you change the version on the PR manually, you will also need to change the name of the version folder.

**Comment by Trenly** [comment]
changing `manifests/o/Ombrelin/PandocGui/1.1/*` to be `manifests/o/Ombrelin/PandocGui/1.1.35/*` to match the version number should fix the error

**Review by Trenly — DISMISSED** [review]

File: manifests/o/Ombrelin/PandocGui/1.1.35/Ombrelin.PandocGui.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /S
```
Comment (OfficialEsco) [inline manifests/o/Ombrelin/PandocGui/1.1.35/Ombrelin.PandocGui.installer.yaml:15]: This is not the Silent install switch for Inno, did they change InstallerType?
```suggestion
InstallerSwitches:
  Silent: /S
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by OfficialEsco** [comment]
Accepted #40111 as that is correct

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**